iotileship/iotile/ship/recipe_manager.py

```diff
@@ -86,6 +86,17 @@ def add_recipe_folder(self, recipe_folder, whitelist=None):
             recipe = RecipeObject.FromArchive(os.path.join(recipe_folder, ship_file), self._recipe_actions, self._recipe_resources)
             self._recipes[recipe.name] = recipe
 
+    def add_recipe_actions(self, recipe_actions):
+        """Add additional valid recipe actions to RecipeManager
+
+        args:
+            recipe_actions (list): List of tuples. First value of tuple is the classname, 
+                second value of tuple is RecipeAction Object
+
+        """
+        for recipe_action in recipe_actions:
```

Comment:
For clarity, it's better to break out the parts of `recipe_action` explicitly here using a destructuring assignment:

```python
for action_name, action in recipe_actions:
    self._recipe_actions[action_name] = action
```

Reply:
done